qiita_pet/test/test_download.py

```diff
@@ -161,21 +161,6 @@ def test_download_study(self):
         response = self.get('/download_study_bioms/1')
         self.assertEqual(response.code, 200)
         exp = (
-            '- 1256812 /protected/processed_data/1_study_1001_closed_'
```

Comment:
Why has this being deleted? I'd to fix this recently by adding this, so I'm surprised now that it needs to be removed.

Reply:
Those lines had a conflict between the master and dev branches, which I fixed by deleting them. BTW the tests are passing.